### ISSUE TITLE
Peer review by aa2686

### ISSUE THREAD

**Issue by avyukt** [SZ559/Analysis-of-Learning-Rate-for-Deep-Neural-Networks#5]
The project aims to study existing algorithms and propose a new method of mathematically identifying an optimal learning rate. Multiple datasets will be used to compare the algorithms and validate the performance and reliability of the algorithms, different tasks. 

What I liked about the project: 
1. Identifying the key factor in training neural networks as the learning rate and aiming to find better ways to optimize it. This is a challenging problem to solve and would have an impact on the time taken to train neural networks. 

2. Discussing the challenges in optimizing learning rate and recent work for the development of algorithms for learning rate.

3. Learning these algorithms to better understand the concepts learned in the class as these would be ways to make them better and address their limitations. The project would definitely help towards learning algorithms at a deeper level. 

What requires more work 
1. Choice of datasets and clear metrics on which to test the algorithms 
It will be helpful to choose a test dataset on which to run and evaluate the algorithms. 
Authors mention different tasks and multiple datasets but there is no mention of any specific application or measure which would be used in this study. 

2. Discussion of challenges in optimizing the learning rate in a specific situation, to illustrate the difficulty in optimizing this rate would be helpful to understand the complexity of the problem and suggest possible improvements and way forward. 

2. The scope of the project seems to be ambitious for a three-month project. This is a challenging problem and the project would be impactful if the learning rate can be optimized for certain specific tasks.
